src/components/Box/__tests__/box-snapshot-tests.jest.js

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Box from "../Box";
+
+/**
+ * There are cases where the component we want to test in the snapshot test will contain additional components.
```

Comment:
this comment is for the developer writing the test, let's remove it before merge